R/glex.R

```diff
@@ -314,6 +314,24 @@ tree_fun_emp <- function(tree, trees, x, all_S, probFunction = NULL) {
   m_all
 }
 
+tree_fun_emp_fastPD <- function(tree, trees, x, all_S) {
+  Tree <- NULL
+  Feature_num <- NULL
+  Feature <- NULL
+  Node <- NULL
+  PathData <- NULL
+  # Calculate matrix
+  tree_info <- trees[Tree == tree, ]
+  tree_info[, Feature := Feature_num - 1L]
+  tree_mat <- tree_info[, .(Tree, Node, Split, Yes, No, Quality, Cover, Feature)]
```

Comment:
I quite dislike that we need to do this workaround by assigning these variables to NULL, since they are really only used to stop the linter from complaining. I think perhaps we should change everything such that we index using strings instead. We can perhaps do something like `tree_info <- trees[get("Tree") == tree, ]`?

Reply:
Yes, `data.table` technically should allow to completely avoid NSE, I'm not sure what the preferred / idomatic approach is (a quick google also suggest using `get` which I guess feels a bit cumbersome but oh well).

`NULL`ing these is of course the quick and dirty workaround and I also dislike it a lot. The other approach would be to chuck a `globalVariables(c(".", "Split", ...))` somewhere in `R/zzz.R` or something but that also has a distinct aftertaste of yikes.